From: Nick Nunley <nicholasx.d.nunley@intel.com>
Date: Wed, 17 Feb 2010 01:05:15 +0000
Subject: igb: remove unused vmolr value
References: FATE#311863 ,bnc#699089
Patch-mainline: v2.6.34-rc1
Git-commit: d3eeb62ddd06ea20a054213cface92d328709285

In an earlier version of igb_write_mc_addr_list() the vmolr register
was modified. This register is no longer accessed, although the variable
still exists. This patch removes it from the function.

Signed-off-by: Nicholas Nunley <nicholasx.d.nunley@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/igb/igb_main.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/net/igb/igb_main.c
+++ b/drivers/net/igb/igb_main.c
@@ -2843,7 +2843,6 @@ static int igb_write_mc_addr_list(struct
 	struct e1000_hw *hw = &adapter->hw;
 	struct dev_mc_list *mc_ptr = netdev->mc_list;
 	u8  *mta_list;
-	u32 vmolr = 0;
 	int i;
 
 	if (netdev_mc_empty(netdev)) {
@@ -2857,9 +2856,6 @@ static int igb_write_mc_addr_list(struct
 	if (!mta_list)
 		return -ENOMEM;
 
-	/* set vmolr receive overflow multicast bit */
-	vmolr |= E1000_VMOLR_ROMPE;
-
 	/* The shared function expects a packed array of only addresses. */
 	mc_ptr = netdev->mc_list;
 
